charts/zenko/values.yaml

```diff
@@ -95,4 +95,22 @@ redis-ha:
     create: true
   replicas:
     servers: *nodeCount
-    sentinels: *nodeCount
\ No newline at end of file
+    sentinels: *nodeCount
+
+grafana:
+  ingress:
+    enabled: false
+    hosts:
```

Comment:
Is this whole `ingress` section required anyway? I'd say no... just use the upstream defaults, no need to override.